deploy/file_list

```diff
@@ -1,8 +1,10 @@
-Assets/Plugins/Android/SoomlaAndroidCore.jar
```

Comment:
@EugeneButusov yes you're right we'll have to check what happens. It might make us go back to writing each file.